packages/cozy-client/src/queries/referencedBy.js

```diff
@@ -0,0 +1,63 @@
+import { IOCozyFile, Doctype, Reference } from '../types'
+
+/**
+ * Checks if the file is referenced by a specific doctype
+ *
+ * @param {IOCozyFile} file - io.cozy.files document
+ * @param {Doctype} referencedBy - Doctype where document is referenced
+ * @returns {boolean} If a reference is found
+ */
+export const isReferencedBy = (file, referencedBy) => {
+  const references =
+    file?.relationships?.referenced_by?.data || file?.referenced_by || []
+
+  return !!references.find(reference => reference.type === referencedBy)
+}
```

Comment:
I don't think queries is the right place to put those helpers. 

What do you think about `/cozyclient/src/associations/helpers.js` ?

Reply:
Indeed, it's better place